include/tscore/ink_inet.h

```diff
@@ -1231,6 +1231,8 @@ struct IpAddr {
                  size_t len  ///< [in] Size of buffer.
   ) const;
 
+  sockaddr *toSockAddr(sockaddr *dest) const;
```

Comment:
Probably good to add a doxygen for this to be consistent with the other functions here.

Reply:
Gah, yeah. Let me fix that.